### ISSUE TITLE
pipenv is painfully slow

### ISSUE THREAD

**Issue by rozsasarpi** [barischrooneyj/bridge-sim#171]
Just installing a new package in the virtual environment, e.g. `pipenv install pypiwin32`, is ridiculously slow.

This seems to be a common problem, see for example https://github.com/pypa/pipenv/issues/2873.

I haven't tried [poetry](https://github.com/python-poetry/poetry) yet but it seems to be promising.

**Comment by barischrooneyj** [comment]
I have mostly switch to poetry, apart from the Docker image. I will be away for a few days, and will finish it off then. If you are not using the Docker image then the git branch `poetry` is worth checking out, maybe you can see if it works for you, and if not what is the error? Instructions are in the `README` file on the branch, though it pretty much comes down to replacing the   `pipenv` command with `poetry`.

**Comment by rozsasarpi** [comment]
Thanks a lot! I just tested `poetry`:

* it works on Win 10, I run `example.py`;
* the install was a lot more pleasant than with `pipenv`. `poetry` showed how many packages are going to be installed and then one-by-one displayed each installed package during the process so at least I knew that it was doing something
* with `poetry` I did not need to manually install `pypiwin32` 👍 

There was just one small thing that was better with `pipenv`: PyCharm automatically found the virtual environment while in case of `poetry` I had to add it manually. Maybe a note could be added to the `README`:

> After installation you can find your `poetry` created virtual environment by using `poetry show -v`. You might need to add the path to `python.exe` manually in your IDE.
